<ui version="4.0" >
 <class>CMilSearchParamsGuiComp</class>
 <widget class="QWidget" name="CMilSearchParamsGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="toolTip" >
        <string>Sets the resolution level for the initial stage (lowest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher first level speeds up the initial search but might make it less reliable because the model might not retain enough distinctive features at a lower resolution.&#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the resolution level for the initial stage (lowest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher first level speeds up the initial search but might make it less reliable because the model might not retain enough distinctive features at a lower resolution.&#xd;
</string>
       </property>
       <property name="text" >
        <string>First Level</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="m_spinFirstLevel" >
       <property name="toolTip" >
        <string>Sets the resolution level for the initial stage (lowest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher first level speeds up the initial search but might make it less reliable because the model might not retain enough distinctive features at a lower resolution.&#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the resolution level for the initial stage (lowest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher first level speeds up the initial search but might make it less reliable because the model might not retain enough distinctive features at a lower resolution.&#xd;
</string>
       </property>
       <property name="maximum" >
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_10" >
       <property name="toolTip" >
        <string>Sets the resolution level for the final stage (the highest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher last level speeds up the initial search but might make it less reliable and less accurate because the model might not retain enough distinctive features at a low resolution. &#xd;
If the specified level is not supported by the search algorithm, the highest acceptable level will be used. &#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the resolution level for the final stage (the highest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher last level speeds up the initial search but might make it less reliable and less accurate because the model might not retain enough distinctive features at a low resolution. &#xd;
If the specified level is not supported by the search algorithm, the highest acceptable level will be used. &#xd;
</string>
       </property>
       <property name="text" >
        <string>Last Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="m_spinSecondLevel" >
       <property name="toolTip" >
        <string>Sets the resolution level for the final stage (the highest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher last level speeds up the initial search but might make it less reliable and less accurate because the model might not retain enough distinctive features at a low resolution. &#xd;
If the specified level is not supported by the search algorithm, the highest acceptable level will be used. &#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the resolution level for the final stage (the highest resolution) of the search.&#xd;
Level 0 is the original target and each higher level is half the size (and resolution) of the previous one. If the specified level is not supported by the search algorithm, the highest possible level will be used. &#xd;
A higher last level speeds up the initial search but might make it less reliable and less accurate because the model might not retain enough distinctive features at a low resolution. &#xd;
If the specified level is not supported by the search algorithm, the highest acceptable level will be used. &#xd;
</string>
       </property>
       <property name="maximum" >
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="toolTip" >
        <string>Sets the algorithm's search speed. &#xd;
Note that increasing the search speed can decrease the robustness &#xd;
and subpixel accuracy of the search operation.&#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the algorithm's search speed. &#xd;
Note that increasing the search speed can decrease the robustness &#xd;
and subpixel accuracy of the search operation.&#xd;
</string>
       </property>
       <property name="text" >
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="m_comboSpeed" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string>Sets the algorithm's search speed. &#xd;
Note that increasing the search speed can decrease the robustness &#xd;
and subpixel accuracy of the search operation.&#xd;
</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the algorithm's search speed. &#xd;
Note that increasing the search speed can decrease the robustness &#xd;
and subpixel accuracy of the search operation.&#xd;
</string>
       </property>
       <item>
        <property name="text" >
         <string>Very High</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>High</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Low</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="toolTip" >
        <string>Sets the accuracy with which to find occurrences. &#xd;
The precision achieved is dependent on the quality of the model and the target. &#xd;
In addition, positional accuracy is slightly affected by the setting of the Speed parameter.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the accuracy with which to find occurrences. &#xd;
The precision achieved is dependent on the quality of the model and the target. &#xd;
In addition, positional accuracy is slightly affected by the setting of the Speed parameter.</string>
       </property>
       <property name="text" >
        <string>Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="m_comboAccuracy" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string>Sets the accuracy with which to find occurrences. &#xd;
The precision achieved is dependent on the quality of the model and the target. &#xd;
In addition, positional accuracy is slightly affected by the setting of the Speed parameter.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the accuracy with which to find occurrences. &#xd;
The precision achieved is dependent on the quality of the model and the target. &#xd;
In addition, positional accuracy is slightly affected by the setting of the Speed parameter.</string>
       </property>
       <item>
        <property name="text" >
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>High</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>Polarity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13" >
         <property name="toolTip" >
          <string/>
         </property>
         <property name="text" >
          <string>Acceptance Target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>Certainty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15" >
         <property name="text" >
          <string>Certainty Target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16" >
         <property name="text" >
          <string>Fit Error Weight</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="m_comboPolarity" >
         <item>
          <property name="text" >
           <string>Same</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Reverse</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Both</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinAcceptanceTarget" >
         <property name="toolTip" >
          <string>Sets the acceptance level for the target score. An occurrence will be returned only if the target score between the target and the model is greater than or equal to this level. &#xd;
The target score is a measure, as a percentage, of edges found in the occurrence that are not found in the original model, weighted by the deviation in position of the common edges.&#xd;
Using M_FIT_ERROR_WEIGHTING_FACTOR, you can specify how much the fit influences the target score.</string>
         </property>
         <property name="whatsThis" >
          <string>Sets the acceptance level for the target score. An occurrence will be returned only if the target score between the target and the model is greater than or equal to this level. &#xd;
The target score is a measure, as a percentage, of edges found in the occurrence that are not found in the original model, weighted by the deviation in position of the common edges.&#xd;
Using M_FIT_ERROR_WEIGHTING_FACTOR, you can specify how much the fit influences the target score.</string>
         </property>
         <property name="suffix" >
          <string> %</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>10</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinCertainty" >
         <property name="suffix" >
          <string> %</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinCertaintyTarget" >
         <property name="suffix" >
          <string> %</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinFitErrorWeight" >
         <property name="suffix" >
          <string> %</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>5</number>
         </property>
         <property name="value" >
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>Nominal Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24" >
         <property name="toolTip" >
          <string>Sets the nominal search angle, relative to the model reference axis angle.</string>
         </property>
         <property name="whatsThis" >
          <string>Sets the nominal search angle, relative to the model reference axis angle.</string>
         </property>
         <property name="text" >
          <string>Nominal Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18" >
         <property name="text" >
          <string>Detail Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19" >
         <property name="text" >
          <string>Smoothness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20" >
         <property name="text" >
          <string>Shared Edges</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21" >
         <property name="text" >
          <string>Target Caching</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QDoubleSpinBox" name="m_spinNominalScale" >
         <property name="minimum" >
          <double>1.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinNominalAngle" >
         <property name="toolTip" >
          <string>Sets the nominal search angle, relative to the model reference axis angle.</string>
         </property>
         <property name="whatsThis" >
          <string>Sets the nominal search angle, relative to the model reference axis angle.</string>
         </property>
         <property name="suffix" >
          <string> Â°</string>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>360</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_comboDetailLevel" >
         <item>
          <property name="text" >
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>High</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Very High</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_spinSmoothness" >
         <property name="suffix" >
          <string> %</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_comboSharedEdges" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Enabled</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_comboTargetCaching" >
         <item>
          <property name="text" >
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Enabled</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_22" >
       <property name="text" >
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_comboFilterType" >
       <item>
        <property name="text" >
         <string>Kernel</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Recursive</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelKernelSize" >
       <property name="text" >
        <string>Kernel Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_spinKernelSize" >
       <property name="minimum" >
        <number>3</number>
       </property>
       <property name="singleStep" >
        <number>2</number>
       </property>
       <property name="value" >
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_11" >
       <property name="toolTip" >
        <string>Sets the maximum search time, in msec.&#xd;
If a search times out, the number of occurrences found at that point will be returned.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the maximum search time, in msec.&#xd;
If a search times out, the number of occurrences found at that point will be returned.</string>
       </property>
       <property name="text" >
        <string>Timeout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_spinTimeout" >
       <property name="toolTip" >
        <string>Sets the maximum search time, in msec.&#xd;
If a search times out, the number of occurrences found at that point will be returned.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the maximum search time, in msec.&#xd;
If a search times out, the number of occurrences found at that point will be returned.</string>
       </property>
       <property name="suffix" >
        <string> ms</string>
       </property>
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="maximum" >
        <number>3000</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
       <property name="value" >
        <number>2000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
